stylesheets/SimpliGov-basic.css

```diff
@@ -396,3 +395,58 @@ h5 {
 	background-color: #ffffff!important;
 	color: #165C96!important;
 }
+/* END Button colors */
+
+/* Button focus and hover */
+/* Submit-Next button focus */
+.form-submit-button:active:focus,
+.form-submit-button:focus,
+.form-section-next:active:focus,
+.form-section-next:focus,
+/* Upload button focus */
+#divWorkflowContent .buttonPopup.blue:active:focus, 
+#divWorkflowContent .buttonPopup.blue:focus {
+    outline: thin dotted!important;

```

Comment:
Just to double-check, are all these `!important` statements necessary? I generally try to avoid using that if at all possible.

Reply:
Agreed, using this isn't good practice. Karen and I have found that if we don't include `!important`, SimpliGov overrides our changes with their default settings. Similarly, Shana has always advised that we include this when she has provided CSS snippets

Reply:
Ok, thanks.